src/libsyntax/feature_gate.rs

```diff
@@ -703,6 +703,8 @@ declare_features! (
     (accepted, extern_crate_self, "1.34.0", Some(56409), None),
     // support for arbitrary delimited token streams in non-macro attributes
     (accepted, unrestricted_attribute_tokens, "1.34.0", Some(55208), None),
+    // add replace and swap functions to RefCell
+    (accepted, refcell_replace_swap, "1.35.0", Some(43570), None),
```

Comment:
These two lines are not necessary as this is a library feature, as opposed to a language feature where it would have been necessary.